postgresql_audit/flask.py

```diff
@@ -6,17 +6,40 @@
 from flask import g, request
 from flask.globals import _app_ctx_stack, _request_ctx_stack
 
-from .base import VersioningManager
+from .base import VersioningManager as BaseVersioningManager
 
 
-class FlaskVersioningManager(VersioningManager):
-    @property
-    def transaction_values(self):
+class VersioningManager(BaseVersioningManager):
```

Comment:
Why change the the name? I'd like to keep the name as ``FlaskVersioningManager`` as its more explicit. Also bear in mind that at one point we can support other frameworks as well (for example Bottle or Pyramid).